app/javascript/react/screens/App/Overview/components/Cards/MigrationPlansCard/MigrationPlansCard.js

```diff
@@ -6,29 +6,35 @@ const MigrationPlansCard = ({
   cardRef,
   showPlanWizardAction,
   showPlanDisabled
-}) => (
-  <div className="card-pf">
-    <div className="card-pf-body">
-      <div className="blank-slate-pf" ref={cardRef}>
-        <EmptyState.Icon />
-        <EmptyState.Title>{__('Migration Plans')}</EmptyState.Title>
-        <EmptyState.Info>
-          {__('Create a migration plan to start migrating.')}
-        </EmptyState.Info>
-        <EmptyState.Action>
-          <Button
-            bsStyle="primary"
-            bsSize="large"
-            onClick={showPlanWizardAction}
-            disabled={showPlanDisabled}
-          >
-            {__('Create Migration Plan')}
-          </Button>
-        </EmptyState.Action>
+}) => {
+  const happyText = __('Create a migration plan to start migrating.');
```

Comment:
i appreciate the humor - but let's use different `const` names ;)

Reply:
Fair enough.. haha. Figured it was clear enough, but I suppose I'll make them more serious.